segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/stat/AutoScaleProcessorTest.java

```diff
@@ -367,4 +393,87 @@ protected void setTimeMillis(long time) {
         }
     }
 
+    @Test
+    public void testCacheGetOnReport() {
+        final int infiniteSize = Integer.MAX_VALUE;
+        final AtomicLong currentTime = new AtomicLong();
+        val evictions = new ArrayList<Map.Entry<Long, Long>>();
+        io.pravega.common.util.SimpleCacheTests.TestSimpleCache<String, Pair<Long, Long>> testSimpleCache =
```

Comment:
Consider using `moq`. This way you won't have to add yet another dependency to this project.

Reply:
Reverted the formatting changes.  Addressed the point on mocks earlier.